next/src/components/_global/sliderComponent/swiperComponent.tsx

```diff
@@ -0,0 +1,125 @@
+'use client';
+
+import 'swiper/css';
+import { useRef, useState } from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { A11y } from 'swiper/modules';
+import styles from './sliderComponent.module.scss';
+
+export default function SwiperComponent({ children, length }: { children: React.ReactNode[]; length: number }) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  const swiperRef = useRef(null);
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access
+  const handlePrev = () => swiperRef.current?.swiper?.slidePrev();
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access
+  const handleNext = () => swiperRef.current?.swiper?.slideNext();
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+  const handleButtonClick = (i) => swiperRef.current?.swiper?.slideToLoop(i);
+  return (
+    <>
+      <Swiper
+        ref={swiperRef}
+        spaceBetween={50}
+        slidesPerView={1.5}
```

Comment:
Here you should declare values which will be visible in mobile view, because we should focus the most on optimizing our code for mobile devices.

Reply:
The values are the same as in design